common/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="TypographyEllipsis">
+    <string name="staking_unstake_all">Unstake all</string>
+
+    <!--    <string name="choose_amount_error_too_big">Sorry, you don\'t have enough funds to spend specified amount</string>-->
+    <string name="choose_amount_error_too_big">Sorry, you don\'t have enough funds to spend the specified amount</string>
+
+    <!--    <string name="staking_setup_amount_too_low">You cannot stake less than the network minimum value (%s)</string>-->
+    <string name="staking_setup_amount_too_low">You can\'t stake less than the minimal value (%s)</string>
+
+    <!--    <string name="add_delegation_missing_account_message">You should add %s account to the wallet in order to delegate</string>-->
+<!--    <string name="staking_missing_account_message">You should add a %s account to your wallet in order to start staking</string>-->
+<!--    <string name="referendum_missing_account_message">You should add %s account to the wallet in order to vote</string>-->
+<!--    <string name="crowdloan_missing_account_message">You should add %s account to the wallet in order to contribute</string>-->
+    <string name="staking_missing_account_message">You need to add a %s account to your wallet in order to start staking</string>
+    <string name="add_delegation_missing_account_message">You need to add a %s account to the wallet in order to delegate</string>
+    <string name="referendum_missing_account_message">You need to add a %s account to the wallet in order to vote</string>
+    <string name="crowdloan_missing_account_message">You need to add a %s account to the wallet in order to contribute</string>
```

Comment:
In the future I want to refactor this to make common part parametric but not now, since it is minor